src/snowflake/cli/api/project/schemas/updatable_model.py

```diff
@@ -155,7 +155,9 @@ def validator_skipping_templated_str(cls, value, handler, info: ValidationInfo):
         setattr(
             cls,
             f"_field_validator_with_verbose_name_to_avoid_name_conflict_{field_name}",
-            field_validator(field_name, mode="wrap")(validator_skipping_templated_str),
+            field_validator(field_name, mode="wrap", check_fields=False)(
```

Comment:
I am curious which field caused this issue?

Reply:
```py
_entity_id: str = PrivateAttr(default=None)
```
Current implementation is assuming every attribute is field, which is not true.

Reply:
I think I fixed it